README.md

```diff
@@ -29,7 +29,9 @@ All of the containers must share a common mount point or data store.  Ironic req
 - final image to be deployed onto node in qcow2 format
 
 The following environment variables can be passed in to customize run-time functionality:
-- PROVISIONING_INTERFACE - interface to use for ironic, dnsmasq(dhcpd) and httpd (default provisioning)
+- MAC_<nodename> - the mac address of the master nodes (used to determine the PROVISIONING_INTERFACE)
+- NODE_NAME - the name of the node that the container is running on
```

Comment:
Why not just provide PROVISIONING_MAC? NODE_NAME does not seem to be required.

Reply:
at the time that we deploy, we don't know what machine the deployment will run on, therefore we don't know what bmh mac to use. So what i am doing is setting the mac of each master and using the node name env, so the container can know which mac to use

Reply:
I do think we could just grep for the mac in all interfaces instead of having the node name.  However, this also works fine.